src/site/content/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Enabling Shared Array Buffer
+authors:
+  - jakearchibald
+description: >
+  The way to enable Shared Array Buffer is changing, but the good news is it works cross-browser and cross-platform.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-TODO-CHANGE-THIS
+date: 2020-12-01
+updated: 2020-12-01
+hero: hero.jpg
+alt: Iframes, images, and windows being accessed by someone 'evil'
+tags:
+  - blog
+  - javascript
+  - shared array buffer
+  - coop
+  - coep
+  - spectre
+  - meltdown
+  - security
+feedback:
+  - api
+---
+
+It's fair to say `SharedArrayBuffer` has had a bit of a rough landing on the web, but things are settling down. Here's what you need to know:
+
+## In brief
+
+- `SharedArrayBuffer` will arrive in Android Chrome 88, but only for cross-origin isolated pages (details below).
+- `SharedArrayBuffer` is currently available in desktop Chrome, but from Chrome 91 it will be limited to cross-origin isolated pages.
+
+You can make a page _cross-origin isolated_ by serving the page with these headers:
+
+```http
+Cross-Origin-Embedder-Policy: require-corp
+Cross-Origin-Opener-Policy: same-origin
+```
+
+Once you do this, your page will not be able to load cross-origin content unless the [`Cross-Origin-Resource-Policy` header](<https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)>) allows it, or if it's requested using [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), and the usual `Access-Control-Allow-*` headers allow it.
```

Comment:
How about:

> Setting these headers means that your page won't have access to cross-origin content unless it allows you to load it, either by setting [`Cross-Origin-Resource-Policy: cross-origin`], or via the existing CORS dance that you know and love (`Access-Control-Allow-*` headers and so forth).

Reply:
I avoided giving a value for CORP, since there are multiple values that might be ok here, like same-site, and I didn't want to have to go into the difference.

I like the bit about acknowledging the joy of CORS. I'll work that in.